src/cdk-experimental/testing/test-app/harnesses/main-component-harness.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness, TestElement} from '../../component-harness';
+import {SubComponentHarness} from './sub-component-harness';
+
+export class MainComponentHarness extends ComponentHarness {
+  readonly title = this.find('h1');
```

Comment:
Should we make these methods? The various elements might not be available when this is created and if it's a getter might end up calling it too often.

Reply:
These actually are methods since the component's `find` method return `() => Promise`